components/homme/src/share/control_mod.F90

```diff
@@ -253,6 +253,21 @@ module control_mod
   real (kind=real_kind), public :: dcmip16_mu_q    = -1       ! additional uniform viscosity (scalar tracers); -1 implies it defaults to dcmip16_mu_s value
   real (kind=real_kind), public :: interp_lon0     = 0.0d0
 
+!PLANAR
+  real (kind=real_kind), public :: Lx, Ly !domain size
+  real (kind=real_kind), public :: Sx, Sy !shift
+  real (kind=real_kind), private, parameter :: tol_zero=1e-10 !tolerance to determine if lx,ly,sx,sy are set
+
+  real (kind=real_kind), public :: bubble_T0 = 270.0       !bubble ref state
+  real (kind=real_kind), public :: bubble_dT = 0.5         !bubble dTheta
+  real (kind=real_kind), public :: bubble_xycenter = 0.0   !bubble xy position
+  real (kind=real_kind), public :: bubble_zcenter = 2000.0  !bubble z position
+  real (kind=real_kind), public :: bubble_ztop = 10000.0   !bubble z top
+  real (kind=real_kind), public :: bubble_radius = 1500.0  !bubble z top
+  logical,               public :: bubble_cosine  = .TRUE. !bubble uniform or cosine
+
+  public :: set_planar_defaults
+
 contains
```

Comment:
I like making these flexible like this. FYI, these constants currently don't agree with either the small bubble from the literature or the large bubble that me & Matt Norman have been running in the CRM. But with this flexibility it doesn't matter! We might want to change the defaults to match one of those, though.